doc/source/changes.rst

```diff
@@ -12,14 +12,29 @@ to adapt in your project configuration, database etc. when upgrading to a new ve
 Version 1.7.2 (CURRENT DEV)
 ---------------------------
 This is the current development version, not yet released.
-Changes thus far are relevant only for users of MapFish Print.
+
+Oereblex related changes
+^^^^^^^^^^^^^^^^^^^^^^^^
+Pyramid Oereb now supports and uses by default geoLink schema version 1.2.0:
+
+ * New doctype 'notice' (will be classified as 'HintRecord'). If you want to add related notice as
+   additional legal provision directly to the public law restriction, You can set the new oereblex
+   'related_notice_as_main' flag in the config of the project.
+ * 'Notice' can have no authority nor enactment_date. In this case, the enactment date will be
+   '01.01.1970' and the authority '-'.
+ * 'Notice' can have no authority_at_web. That was not supported by MapFish Print. If you use MapFish Print
```

Comment:
Please move this to the "MapFish Print related changes" section

Reply:
MFP section as already a note (without oereblex mention).
This mention is for user using oereblex (with MFP). So I let it there.